web/cobrands/sass/_base.scss

```diff
@@ -1451,63 +1451,58 @@ html.js #map .noscript {
   #{$left}: 0;
   #{$right}: 0;
   bottom: 0;
-  background:#333;
-  background:rgba(0, 0, 0, 0.7);
-  margin:0;
+  background: #333;
+  background: rgba(0, 0, 0, 0.7);
+  margin: 0;
+
   a {
-    @include inline-block;
-    font-size:0.6875em;
-    color:#fff;
-    padding: flip(0.6em 3em 0.5em 1em, 0.6em 1em 0.5em 3em);
-    background-repeat:no-repeat;
-    &#hide_pins_link {
-      background-image:url($image-sprite);
-      background-position: flip(right, -341px) -3976px;
-    }
-    &#map_permalink {
-      background-image:url($image-sprite);
-      background-position: flip(right, -341px) -4070px;
-    }
-    &#fms_shortlist_all {
-      padding: 0.6em 1em 0.5em 1em;
-      &:after {
-         content: "";
-         display: inline-block;
-         width: 16px;
-         height: 16px;
-         @include svg-background-image('/cobrands/fixmystreet/images/map-marker');
```

Comment:
map-marker image can be removed then? Was it ever used?

Reply:
Done!